CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 1. [#2842](https://github.com/influxdata/chronograf/pull/2842): Use Generic APIKey for Oauth2 group lookup
 1. [#2850](https://github.com/influxdata/chronograf/pull/2850): Fix bug in which resizing any cell in a dashboard causes a Gauge cell to resize
 1. [#2851] (https://github.com/influxdata/chronograf/pull/2851): Maintain y axis labels in dashboard cells
+1. [#2819] (https://github.com/influxdata/chronograf/pull/2819): Fix error when using --new-sources from CLI and deprecating --new-sources
```

Comment:
could you clarify this changelog message? it gives me the impression that i can both _use_ `--new-sources` and that it's been deprecated, which are contradictory

Reply:
Yea. There was an error when using `--new-sources`, which that PR fixed as well as added the deprecation log which is what that message was trying to convey.

Reply:
ah, i see. so how about:
Fix error message when attempting to use now-deprecated `--new-sources` CLI flag
?